packages/client/src/contents/__fixtures__/commercepages.fixtures.js

```diff
@@ -1,48 +1,27 @@
-import join from 'proper-url-join';
-import moxios from 'moxios';
+import { rest } from 'msw';
 
-/**
- * @typedef {object} RequestParams
- * @property {object} query - Query to request.
- * @property {Array} [response] - Contents payload.
- */
+const path = '/api/content/v1/commercepages';
 
 /**
  * Response payloads.
  */
 export default {
   get: {
     /**
-     * Success moxios request.
+     * Success msw request.
      *
-     * @param {RequestParams} params - Params to moxios request.
+     * @param {Array} response - Commerce Pages payload.
```

Comment:
Response seems to be an object and not an array. Can you check if that is the expected type?

Reply:
Done.